download.md

```diff
@@ -4,74 +4,6 @@ title: Puppy Linux Downloads
 ---
 ## Puppy Linux Downloads
 
-### These are the latest woof-CE built puppies and their discussion thread
+Visit our [download](https://forum.puppylinux.com/puppy-linux-collection) page on the Forum
```

Comment:
Why not just delete download.md and make the top bar point to the forum page directly?

Reply:
The top bar does point directly........

I think the download.md still needs to be there although no longer linked to the rest of the site in case there are links out in the wild.... e.g. in /rootfs-skeleton/usr/share/doc/release-skeleton.bottom.htm

Reply:
That can be fixed using a HTML redirect.

Reply:
I know how to redirect a page..... I don't know how to do it on GitHub...........